data/event-schemas/schema/m.room.power_levels.yaml

```diff
@@ -5,6 +5,11 @@ description: |-
   This event specifies the minimum level a user must have in order to perform a
   certain action. It also specifies the levels of each user in the room.
 
+  The range of power levels is specified by the matrix-specific Canonical JSON
+  specification (https://spec.matrix.org/v1.3/appendices/#canonical-json), which
+  allows a range of `[-(2**53)+1, (2**53)-1]` for integers. Leading zeros are
+  not allowed.
```

Comment:
I don't see where leading zeros are banned?

Reply:
That's my interpretation of

> int       = %x30 / ( %x31-39 *digit )

in the [grammar](https://spec.matrix.org/v1.3/appendices/#grammar). But maybe I'm mistaken?

Reply:
As per https://datatracker.ietf.org/doc/html/rfc7159#section-6

> Leading zeros are not allowed.

As the purpose of this PR is to clarify the range, I think the fact about leading zeros fits there - as the grammar itself explains it via the the `int = ` notation.